prep-css-display/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<head>
+  <title>CSS Display Properties</title>
+  </head>
+    <div> I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+  <style>
+    div{
+      border:4px solid blue;
+    }
+  </style>
+  <div>
+    I am a div but I have <span>span elements inside me</span>.
+  Span elements are <span>display inline by default</span>,
+  so they only take up as much as space as they need.
+  </div>
+  <style>
+span{
+  border: 4px solid green;
+  height: 100px;
+  width:100px;
+}
+  </style>
+  <div>
+    <button>Hello!</button>
+    <button>Buttons are</button>
+    <button>Inline Block</button>
+    <button>By Default</button>
+  </div>
+  <style>
+    button{
+      border:4px solid red;
+      height:40px;
+      width:200px;
+    }
+  </style><ul>
+  <li>
+    Home
+  </li>
+  <li>
+    About us
+  </li>
+  <li>
+    Contact Us
+  </li></ul>
+  <style>
+ ul{
+  border: 4px solid purple;
+}
+li{
+  border:4px solid magenta;
+  display: inline;
+}
```

Comment:
Hi Gabrielle, for this exercise, we want to write all of our CSS code inside one <style> element in the <head> of the document.